sr_port/mdef.h

```diff
@@ -270,32 +271,32 @@ typedef UINTPTR_T uintszofptr_t;
 #define NON_X86_64_ONLY(x)    		x
 #endif /* __x86_64__ */
 
-#ifdef __armv7l__
-#define ARMV7L_ONLY(x)			x
-#define NON_ARMV7L_ONLY(x)
-#define	X86_64_OR_ARMV7L_ONLY(x)	x
+#if defined(__armv6l__) || defined(__armv7l__)
+#define ARM32_ONLY(x)			x
+#define NON_ARM32_ONLY(x)
+#define	X86_64_OR_ARM32_ONLY(x)		x
```

Comment:
What is this line's purpose here when X86_64_OR_ARM32_ONLY apparently gets reset a few lines down?

Reply:
No purpose. Removed.